vtiger_connector_calendar/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
 {
-    'name': 'Calendar VTiger Integration',
-    'version': '10.0.1.0.0',
-    'summary': 'Calendar VTiger Integration',
-    'description': """
-        Calendar VTiger Integration""",
+    'name': 'VTiger Base Connector Calender',
```

Comment:
BASE nathi aa.
This module depends on base... its a VTiger Calendar Integration

Reply:
Yes, Yes, okay sir, I will correct it.
Thanks